### DISCUSSION TITLE
Cannot see the pagination when using table block

### DISCUSSION THREAD

**Discussion by ndlang** [Budibase/budibase#6224] [Help]
![image](https://user-images.githubusercontent.com/17033357/172423595-a754c84c-2890-4ecd-a97c-5de0478972c2.png)

**Comment by melohagan** [comment]
Hi @ndlang 

Thanks for posting, although there a couple of open issues around this:

- https://github.com/Budibase/budibase/issues/3974
- https://github.com/Budibase/budibase/issues/6226

**Reply by ndlang** [reply]
Thank you for the update. Any Idea on when these issues will be resolved?
